### PR TITLE
Feature/python compiler

### PR DESCRIPTION
**Pull request by andres-cox**
It is added python3.9 binaries and a small example.

Paths for both client_file and binaries need to be refactor if server_file.py is moved

### CODE REVIEW

File: code_editor/core/compile/compiler.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#
+# @javacompiler.py Copyright (c) 2020 Jalasoft.
```
Comment (alvaro-cruz) [inline code_editor/core/compile/compiler.py]: @javacompiler.py must be the name of the file

File: code_editor/core/compile/parameters.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#
+# @javacompiler.py Copyright (c) 2020 Jalasoft.
```
Comment (alvaro-cruz) [inline code_editor/core/compile/parameters.py]: must be the name of the file

File: code_editor/core/server_file.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#
+# @javacompiler.py Copyright (c) 2020 Jalasoft.
```
Comment (alvaro-cruz) [inline code_editor/core/server_file.py]: Must be the name of the file

**Review by alvaro-cruz — CHANGES_REQUESTED** [review]

**Comment by sontiveros-AT** [comment]
The only problem with the .py files is the copyright detail, where it should refer to the file itself instead of java_compiler.py in all of them. After fixing that we can merge with the develop branch.

**Comment by sontiveros-AT** [comment]
Please don't include .idea folder.

**Pull request closed without merging**